subworkflows/nf-core/fastq_extract_kraken_krakentools/meta.yml

```
@@ -0,0 +1,50 @@
+name: "fastq_extract_kraken_krakentools"
+description: Extract classified Kraken2 output by taxonomic id
+keywords:
+  - classify
+  - metagenomics
+  - fastq
+  - db
+  - kraken2
+  - krakentools
+  - extract
+components:
+  - kraken2/kraken2
+  - krakentools/extractkrakenreads
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - ch_reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+  - ch_db:
+      type: directory
+      description: Kraken2 database
+  - val_taxid:
+      type: string
+      description: A list of taxid separated by spaces
```

```suggestion
      description: A string of pne or more of taxid separated by spaces
```

I worry someone might read 'list' as a Groovy list...